openedx/core/djangoapps/oauth_dispatch/tests/test_clear_expired_tokens.py

```diff
@@ -0,0 +1,63 @@
+from datetime import timedelta
```

Comment:
Tests should reside near the code being tested. This file belongs in `openedx/core/djangoapps/oauth_dispatch/management/commands/tests/test_edx_clear_expired_tokens.py`